package.json

```
@@ -12,7 +12,7 @@
     },
     "icon": "deepscan_icon.png",
     "engines": {
-        "vscode": "^1.25.0"
+        "vscode": "^1.53.0"
```

```suggestion
        "vscode": "^1.56.0"
```

client/package.json 에도 engines.vscode 속성 있는데 같이 수정해 주세요.